Vagrantfile

```diff
@@ -16,8 +16,8 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.vm.box = BOX
   config.vm.box_download_insecure = true
   config.vm.box_check_update = false
-  config.vm.synced_folder '.', '/orchestrator', type: 'rsync',
-    rsync__auto: true
+  config.vm.synced_folder '.', '/orchestrator'
+  #, type: 'rsync', rsync__auto: true
```

Comment:
Can you please explain this change?

Reply:
rsync__auto want work on my Vagrant environment ;(
when i change some sources on host OS, it's not rsynced into guest maching
ok. i rollback this change

Reply:
Sorry, please elaborate. Did you introduce `rsync__auto` in the first place?

Reply:
i try use following variants
1)
```
 config.vm.synced_folder '.', '/orchestrator', type: 'rsync'
```

it's require run ```vagrant reload``` when i change any file on my host machine
2)
```
 config.vm.synced_folder '.', '/orchestrator', type: 'rsync',  rsync__auto: true
```
it not worked, and i don't investigate why
then i just commented this options and default sharing mode over vboxfs works as i expected